content/en/bits_ai/query_examples.md

```diff
@@ -42,17 +42,14 @@ Bits AI enables natural language querying for service health and ownership of yo
 `Show me traces for web-store that are slower than 1s`:
 {{< img src="bits_ai/query_examples/slow-web-store-traces.png" alt="Query result for slow traces" style="width:90%;">}}
 
-## Infrastructure data (Inventories SQL)
+## Infrastructure data (DDSQL Editor)
 
-<div class="alert alert-info">Bits AI includes access to Inventories SQL, which is in private beta.</div>
+<div class="alert alert-info">Bits AI includes access to [DDSQL Editor][1], which is in private beta.</div>
 
-Infrastructure resource data can be queried in [Inventories SQL][1].
+`Most Common Instance Types`:
+{{< img src="ddsql_editor/query-ui-overview.png" alt="Query result for most common instance types" style="width:90%;">}}
```

Comment:
Is this the right way to link to this image? I want to reuse the image in this section https://docs.datadoghq.com/ddsql_editor/#overview

Reply:
Yes, that is the correct way to link to images. 

You can see a staged build of the doc if you name you branch following the format `SLACK_USERNAME/BRANCH_NAME`. As described in the Contributor guidelines:
https://github.com/DataDog/documentation#datadog-staff

FYI - Using an image from another doc can cause problems when/if the image is updated by someone else.